packages/next/src/utils.ts

```diff
@@ -2352,14 +2352,16 @@ interface MiddlewareManifestV2 {
   functions?: { [page: string]: EdgeFunctionInfoV2 };
 }
 
+type Regions = 'home' | 'edge' | 'auto' | string[] | 'all' | 'default';
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Can `'global'` be a value here too or does it only show up in the `string[]`?